src/components/Link.tsx

```diff
@@ -43,32 +50,47 @@ const Link: React.FC<LinkProps> = props => {
     underline = false,
     onClick,
     disabled,
+    href,
     ...linkProps
   } = props;
 
   const classes = useStyles(props);
 
-  return (
-    <Typography
-      component="a"
-      className={classNames(className, {
-        [classes.root]: true,
-        [classes[color]]: true,
-        [classes.underline]: underline,
-        [classes.disabled]: disabled
-      })}
-      onClick={event => {
-        if (disabled) {
-          return;
-        }
+  const commonLinkProps = {
+    className: classNames(className, {
+      [classes.root]: true,
+      [classes[color]]: true,
+      [classes.underline]: underline,
+      [classes.noUnderline]: !underline,
+      [classes.disabled]: disabled
+    }),
+    onClick: event => {
+      if (disabled || !onClick) {
+        return;
+      }
 
-        event.preventDefault();
-        onClick();
-      }}
-      {...linkProps}
-    >
-      {children}
-    </Typography>
+      event.preventDefault();
+      onClick();
+    },
+    ...linkProps
+  };
+
+  return (
+    <>
+      {!!href && !isExternalURL(href) ? (
+        <RouterLink to={disabled ? undefined : href} {...commonLinkProps}>
```

Comment:
`commonLinkProps` has `onClick` that `Link` doesn't have in props. It doesn't cause any weird behaviour?

Reply:
Link extends `a` tag props which has an onClick property